frontends/infinite-corridor/src/pages/HomePage.tsx

```diff
@@ -141,6 +236,92 @@ const HomePage: React.FC = () => {
           </div>
         </Grid>
       </GridContainer>
+      <TabContext value={tabValue}>
+        <GridContainer>
+          <GridColumn variant="single-full">
+            <h3>Upcoming Courses</h3>
+            <TabList className="courses-nav" onChange={handleChange}>
+              <Tab label="All" value="all" />
+              <Tab label="Certificate" value="certificate" />
+              <Tab label="Micromasters" value="micromasters" />
+              <Tab label="Professional ed" value="professional" />
+            </TabList>
+            <TabPanel value="all" className="courses-carusel">
+              <HomePageCarousel
+                query={upcomingCourseQuery}
+                onActivateCard={activateResource}
+              />
+            </TabPanel>
+            <TabPanel value="certificate">
```

Comment:
**Comment 1:** Only the first tab has class `"courses-carusel"`. Seems like the others should, too? *Or maybe something similar can go on TabList*.

**Comment 2:** There is an annoying issue here... Try clicking the prev/next buttons when on "All" tab and they do not work. They are obscured by `"MuiTabs-root"` div (added blue border):

<img width="1229" alt="Screenshot 2023-03-31 at 2 35 43 PM" src="https://user-images.githubusercontent.com/9010790/229202421-fc269d7e-56c1-4b6a-a6a7-a73709b99148.png">

Avoiding this issue was the original point of TitledCarousel: my goal there was a component that encapsulates the `nuka-carousel` component itself and the prev/next button logic. (*I didn't really care about the title, it was just convenient based on the designs to put it in that component since it was natural to put the buttons in the same DOM element as the title.*) 

I can think of a few options:
1. Change the width of `.MuiTabs-root`
2. Refactor TitledCarousel so that the buttons are more flexible. E.g: assign a ref to the title element, pass the ref to TitledCarousel, and use [`createPortal`](https://react.dev/reference/react-dom/createPortal) in TitledCarousel to render the buttons inside the next/prev title area. 
3. Move the tabs into the Carousel component. _Seems to make the carousel component too complicated._

(1) is easiest. Conceptually, I like (2). Whatever you think is best...

Reply:
Since the design will likely change, I just did 1